<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupsAdminForm</class>
 <widget class="QWidget" name="GroupsAdminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonNew">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add New Group</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Remove Group</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>486</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Groups</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Group Attributes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Group Members</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Full Users List</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidgetGroups">
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="3" colspan="2">
    <widget class="QTableWidget" name="tableWidgetGroupAttribute">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3" rowspan="3">
    <widget class="QListWidget" name="listWidgetGroupUsers">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButtonAddUsers">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/da_galaxy.qrc">
       <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="3">
    <widget class="QListWidget" name="listWidgetAllUsers">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButtonRemoveUsers">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/da_galaxy.qrc">
       <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/da_galaxy.qrc"/>
 </resources>
 <connections/>
</ui>
